apollo-ast/src/main/kotlin/com/apollographql/apollo3/ast/internal/ExecutableValidationScope.kt

```diff
@@ -70,6 +75,11 @@ internal class ExecutableValidationScope(
       it.validate()
     }
 
+    deferDirectiveLabels.clear()
+    operations.forEach {
+      it.validateDeferDirectives()
```

Comment:
Could you plug into the more generic `validate()` above (and pass a path along the way). No strong preference but the other validations use that and it saves traversing the operation mutiple times. Not sure if that's a real concern or not. Just thought I'd bring it up